rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -24,6 +26,21 @@
             </div>
         {% endif %}
         <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl" rel="noopener noreferrer">OpenKAT</a></span>
+        {% if show_report_types %}
+            <p>
+                {% for plugin_id, report_types in plugin_report_types.items %}
+                    {% if plugin_id == plugin.id %}
+                        {% if report_types|length > 1 %}
```

Comment:
See [this](https://peps.python.org/pep-0008/): For sequences, (strings, lists, tuples), use the fact that empty sequences are false:
```python3
# Correct:
if not seq:
if seq:
# Wrong:
if len(seq):
if not len(seq):
```
```suggestion
                        {% if report_types %}
```

Reply:
No longer applicable, code has been changed/removed